File: 프로그래머스/lv2/12945. 피보나치 수/피보나치 수.py
def solution(n):
    answer = 0
    fibo = [0] * (n + 1)
    fibo[0] = 0
    fibo[1] = 1

    for idx in range(2, n+1):
        fibo[idx] = fibo[idx-1] + fibo[idx-2]
    answer = fibo[-1] % 1234567
    return answer

print(solution(3))

print(solution(5))

File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.js
function solution(n) {
    var answer = 0;
    let fibo = new Array(n + 1).fill(0);
    fibo[0] = 0
    fibo[1] = 1
    for (let i = 2; i < n + 1; i++) {
        fibo[i] = (fibo[i-1] + fibo[i-2]) % 1234567
    }
    console.log(fibo)
    return fibo[fibo.length - 1];
}